### PR TITLE
Updates for AAPS Settings Exports

### PR DESCRIPTION
**Pull request by vanelsberg**
@emilisev Review requested?

Draft explaining settings exports and using in automations (recommended).
Not sure about the use internal references?

### CODE REVIEW

File: docs/EN/DailyLifeWithAaps/Automations.md [hunk 1]
```diff
@@ -2,14 +2,27 @@
...
+Automations performs specific actions based on one or more conditions or triggers. Triggers can include irregular events like low or high blood glucose (BG) levels, or a set amount of negative insulin on board (IOB). Automations can also handle recurring events, such as meals or exercise at certain times of day, or when the user is within a specific distance of a GPS location or a WIFI SSID area. Automation can execute AAPS settings backups based on a schedule or on every Pod change.
 
-There are a wide range of **Automation** options, and users are encouraged to study these within the **AAPS** app, in the **Automation** section. You can also search the **AAPS** user groups on **Facebook** and **Discord** for **Automation** examples from other users. 
+Automations rules are created and modified from the Automations tab. Each rule is defined by two properties:
+
+- One or more conditions or 'triggers' that start an action.
+
+    Think of a certain time schedule, an event or propertie value in AAPS
```
Comment (emilisev) [inline docs/EN/DailyLifeWithAaps/Automations.md:13]: property

File: docs/EN/DailyLifeWithAaps/Automations.md [hunk 2]
```diff
@@ -2,14 +2,27 @@
...
+
+- One or more actions to perform.
+
+    Such as an alarm or settings a profile percentage or exporting the AAPS settings on Pod change.
+
+
+There are a wide range of Automation options, and users are encouraged to study these within the AAPS app, in the Automation section. You can also search the AAPS user groups on **Facebook** and **Discord** for Automation examples from other users.
```
Comment (emilisev) [inline docs/EN/DailyLifeWithAaps/Automations.md:20]: **Facebook** and **Discord** -> link to [**Facebook** and **Discord**](../GettingHelp/WhereCanIGetHelp.md)

File: docs/EN/Maintenance/ExportImportSettings.md [hunk 3]
```diff
@@ -1,32 +1,62 @@
 # Creating and restoring back-ups
 
-It is important to keep back-ups of the items listed below. Best practice is to keep a back-up on a local hard drive as well as a cloud location (i.e. Google Drive, box, etc…). Below are the items that it is recommended to keep back-ups of.
+When installing AndroidAPS on your phone it becomes a "medical device" you are depended on. It is recommended to have an
+emergency backup plan for when your phone gets defective, stolen or lost. Therefore, it is essential to prepare by asking yourself, "what if? 
+
+To enable restoring your AndroidAPS setup to an existing or new phone, it is important to keep following items in a safe place (read: not on your phone).
+Best practice is to keep backups on a local hard drive, USB stick or (preferred) cloud storage like Google Drive, Microsoft 365 OneDrive.
+By storing your backups in the cloud you'll always have everything needed accessible form your phone to restore anywhere and anytime.
```
[the remark this answers is not in the document]
Comment (vanelsberg) [reply docs/EN/Maintenance/ExportImportSettings.md:8]: Yes. General rule is to always have an "offline" backup of the backup. Cloud backup is prefrerred.

**Comment by vanelsberg** [comment]
> @emilisev @vanelsberg is that ready to merge?

I did update for @emilisev 's comments. So for me, yes, ready to merge.

**Pull request merged**